### PR TITLE
Use miniconda3

### PR DESCRIPTION
**Pull request by pelson**
Fixes https://github.com/conda-forge/staged-recipes/issues/215

### CODE REVIEW

**Comment by pelson** [comment]
> I was trying to look for recipes to lint for you, but couldn't find any.

Clever little fellow, isn't he. :wink:

File: .CI/build_all [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 
+# Install miniconda
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh
+bash miniconda.sh -b -p ~/miniconda
 
-# Set the numpy variable. This isn't used, but conda-build complains if we haven't set it already.
-export CONDA_NPY=19
-
-wget https://raw.githubusercontent.com/pelson/Obvious-CI/master/bootstrap-obvious-ci-and-miniconda.py
-python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 2 --without-obvci && source ~/miniconda/bin/activate root
```
Comment (pelson) [inline .CI/build_all]: Woops. Probably need the source activate still...

**Comment by jakirkham** [comment]
Is the the linter bot on Heroku?

**Comment by pelson** [comment]
Despite the fact that the little icon says that one travis job runs on linux and the other on OSX, I got the following message (on the apparently Linux system):

```
cannot execute native linux-64 binary, output from 'uname -a' is:
Darwin Traviss-Mac-408.local 13.4.0 Darwin Kernel Version 13.4.0: Sun Aug 17 19:50:11 PDT 2014; root:xnu-2422.115.4~1/RELEASE_X86_64 x86_64
```

As a result, I've updated the PR to simply use miniconda3.

**Comment by jakirkham** [comment]
Added fixes note to PR. :smile:

**Pull request merged**